core/src/autogluon/core/utils/try_import.py

```diff
@@ -45,34 +45,41 @@ def try_import_mxnet():
 def try_import_catboost():
     try:
         import catboost
+    except ImportError as e:
+        raise ImportError("`import catboost` failed."
+                          "A quick tip is to install via `pip install catboost`.")
     except ValueError as e:
         raise ImportError("Import catboost failed. Numpy version may be outdated, "
-                          "Please ensure numpy version >=1.16.0. If it is not, please try 'pip uninstall numpy; pip install numpy>=1.17.0' Detailed info: {}".format(str(e)))
+                          "Please ensure numpy version >=1.16.0. If it is not, please try 'pip uninstall numpy -y; pip install numpy>=1.17.0' "
```

Comment:
Just noticed: `==1.19.5`

Reply:
Will keep as is for now since this is specific to catboost's minimum numpy version. Not sure the context of when this error used to occur, but doubt it will ever be triggered at this point since we require more recent numpy version.